accio-tests/src/test/java/io/accio/testing/bigquery/AbstractCacheTest.java

```diff
@@ -123,4 +139,27 @@ protected List<Object[]> queryBigQuery(String statement)
             throw new RuntimeException(e);
         }
     }
+
+    private void waitCacheReady(String catalog, String schema, String name)
+    {
+        try {
+            CompletableFuture.runAsync(() -> {
+                try {
+                    while (true) {
+                        Optional<CacheInfoPair> cachedTable = Optional.ofNullable(cachedTableMapping.get().getCacheInfoPair(catalog, schema, name));
+                        if (cachedTable.isPresent()) {
+                            break;
+                        }
+                        Thread.sleep(1000);
+                    }
+                }
+                catch (Exception e) {
+                    throw new RuntimeException(e);
+                }
+            }).get(30, TimeUnit.SECONDS);
+        }
+        catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
```

Comment:
Add a static method to catch Exception then throw RuntimeException is a good idea?